api-cpp-grpc/src/UserGRPC.cpp

```diff
@@ -21,6 +22,26 @@ using dfx::api::UserAPI;
 using namespace dfx::api::grpc;
 using namespace ::grpc;
 
+// Helpers for Birth Date which in gRPC is a EPOCH time, but WebSocket/REST is a %F string.
+// Internally, Cloud API treats birth date as a string.
+std::string epocToString(uint64_t when)
+{
+    time_t epochtime = when;
+    struct tm date;
+    date = *std::localtime(&epochtime);
+    char buffer[256];
+    strftime(buffer, sizeof(buffer), "%F", &date); // %F = Short YYYY-MM-DD, equivalent to %Y-%m-%d  2001-08-23
+    return std::string(buffer);
+}
+
+uint64_t strToEpoch(const std::string& str)
```

Comment:
Need to fix this to use something like C++11 get_time from <iomanip> so doesn't fail on Windows